components/motor/esp_simplefoc/CMakeLists.txt

```diff
@@ -6,11 +6,12 @@ set(SRC_FILES
     "port/esp/esp_app_fmath.cpp"
     "port/esp/esp_app_print.cpp"
     "port/angle_sensor/as5600.cpp"
-    "port/angle_sensor/mt6701.cpp")
+    "port/angle_sensor/mt6701.cpp"
+    "port/angle_sensor/as5048a.cpp")
 
 set(INC_FILES "port/esp/include/" "port/angle_sensor")
 set(IGNORE_FILES "port/esp/esp_hal_adc.cpp" "port/esp/esp_hal_mcpwm.cpp"
-                 "port/esp/esp_hal_bldc_3pwm.cpp" "port/angle_sensor/mt6701.cpp")
+                 "port/esp/esp_hal_bldc_3pwm.cpp" "port/angle_sensor/mt6701.cpp" "port/angle_sensor/as5048a.cpp")
```

Comment:
@demianzenkov  Please run `pre-commit install` under this project. Then during `git commit` stage, it should auto-find and modify the format or spelling issues.

As you have already added commits, please run `SKIP=no-commit-to-branch pre-commit run --files components/motor/esp_simplefoc` to manually run pre-commit on the files you have committed

Reply:
```
demian@Demians-MacBook-Pro-M1-Home esp-iot-solution % SKIP=no-commit-to-branch pre-commit run --files components/motor/esp_simplefoc
Check copyright notices..............................(no files to check)Skipped
file contents sorter.................................(no files to check)Skipped
trim trailing whitespace.............................(no files to check)Skipped
fix end of files.....................................(no files to check)Skipped
check that executables have shebangs.................(no files to check)Skipped
check that scripts with shebangs are executable......(no files to check)Skipped
mixed line ending....................................(no files to check)Skipped
fix double quoted strings............................(no files to check)Skipped
Do not use more than one slash in the branch name.......................Skipped
Do not use uppercase letters in the branch name.........................Skipped
astyle formatter.....................................(no files to check)Skipped
Check File Permissions...............................(no files to check)Skipped
Validate executable-list.txt.............................................Passed
codespell............................................(no files to check)Skipped
```

Reply:
@demianzenkov Sorry for the Incorrect command, it seems the `pre-commit` skipped all checks. 

Please run `SKIP=no-commit-to-branch pre-commit run --files components/motor/esp_simplefoc/**/*`

Reply:
If it works as normal, https://github.com/espressif/esp-iot-solution/pull/382#discussion_r1677262648 will be fixed automaticly